static/js/reducers/index.js

```diff
@@ -44,6 +47,34 @@ export const courseList = (state = INITIAL_COURSE_LIST_STATE, action) => {
   }
 };
 
+const INITIAl_USER_IMAGE_STATE = {
+  imageUrl: SETTINGS.edx_base_url + 'static/images/profiles/default_120.png'
```

Comment:
Can you investigate if there's a way to join these two pieces so that it doesn't matter if `SETTINGS.edx_base_url` has the trailing slash or not? You may want to start here: http://stackoverflow.com/questions/29855098/is-there-a-built-in-javascript-function-similar-to-os-path-join